routes/users.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router = express.Router();
+const data = require('../data');
+const userData = data.users;
+
+/*
+NOTE: REPLACE IF (TRUE) IN ALL THESE WITH THE SESSION EVENTUALLY
+ALSO DO ERROR CHECKING IN EVERY FUNCTION
+*/
+router.get('/:userid', async (req, res) => {
+    // Show profile information
+    try {
+        if (true){
```

Comment:
We have to validate if the provided userId is a valid ObjectId or not and should return 400 - Bad Request in case of invalid userId.